lib/cli.js

```diff
@@ -123,6 +123,14 @@ function init() {
       return;
     }
 
+    if (cmd.startsWith('generator-')) {
```

Comment:
Agreed. We should still warn though, so the user learns the correct way of doing it.